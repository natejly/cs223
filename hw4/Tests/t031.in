P1,2,3,1,3
P2,2,3,2,2
P3,2,1,6,0
P4,0,0,0,9

P1 P2
P1 P3
P2 P3
P1 P4
P4 P3
