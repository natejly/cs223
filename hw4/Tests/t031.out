P1 5.5 - P2 4.5
P1 6.5 - P3 3.5
P2 6.5 - P3 3.5
P1 6.0 - P4 4.0
P3 6.0 - P4 4.0
